components/badge.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../style.css">
+    <script src="https://kit.fontawesome.com/7fa15c8e24.js" crossorigin="anonymous"></script>
+    <title>Badge | Crypto UI</title>
+</head>
+<body>
+    <div class="p-s container">
+
+        <nav class="navigation">
+            <div class="nav-left">
+                <a href="../index.html" class="txt-l fw-black p-s m-s">Crypto&nbsp;UI</a>
+            </div>
+            <div class="nav-right">
+                <a href="https://github.com/harshitadaswani" target="_blank" rel="noopener noreferrer" class="txt-l fw-black p-s m-s"><i class="fab fa-github-square icons"></i></i></a>
+                <a href="https://www.linkedin.com/in/harshitabdaswani/" target="_blank" rel="noopener noreferrer" class="txt-l fw-black p-s m-s"><i class="fab fa-linkedin icons"></i></a>
+                <a href="https://twitter.com/harshi_daswani" target="_blank" rel="noopener noreferrer" class="txt-l fw-black p-s m-s"><i class="fab fa-twitter-square icons"></i></a>
+            </div>
+        </nav>
+
+            <aside>
+                <div class="list-topic fw-semibold">
+                    <a href="../docs/introduction.html">Getting Started</a>
+                </div>
+                <ul class="list-subtopic">
+                    <li class="p-xs m-xs"><a href="../docs/introduction.html">Introduction</a></li>
+                    <li class="p-xs m-xs"><a href="../docs/typography.html">Typography</a></li>
+                    <li class="p-xs m-xs"><a href="../docs/colors.html">Colors</a></li>
+                </ul>
+
+                <div class="list-topic fw-semibold">
+                    <a href="avatar.html">Components</a>
+                </div>
+                <ul class="list-subtopic">
+                    <li class="p-xs m-xs"><a href="avatar.html">Avatar</a></li>
+                    <li class="p-xs m-xs"><a href="alert.html">Alert</a></li>
+                    <li class="p-xs m-xs li-active fw-bold"><a href="badge.html">Badge</a></li>
+                    <li class="p-xs m-xs"><a href="button.html">Button</a></li>
+                    <li class="p-xs m-xs"><a href="card.html">Card</a></li>
+                    <li class="p-xs m-xs"><a href="image.html">Image</a></li>
+                    <li class="p-xs m-xs"><a href="input.html">Input</a></li>                    
+                </ul>
+
+            </aside>
+
+        <main class="main-container">
+            <h1 class="h1-title">Badge</h1>
```

Comment:
if h1-title gives colors name it accordingly

Reply:
h1-title is a utility CSS class having styling related to the title elements